src/storage/src/hummock/compactor/mod.rs

```diff
@@ -284,6 +419,7 @@ pub fn start_compactor(
     hummock_meta_client: Arc<dyn HummockMetaClient>,
     sstable_object_id_manager: Arc<SstableObjectIdManager>,
     compaction_catalog_manager_ref: CompactionCatalogManagerRef,
+    compactor_type: CompactorType,
```

Comment:
how about reuse the CompactorMode?

Reply:
We can follow the design of this [PR](https://github.com/risingwavelabs/risingwave/pull/21818/files) and directly determine the startup mode via `CompactorMode`.